index.html

```diff
@@ -0,0 +1,1975 @@
+  <main>
+    <article class="section-2">
+      <h4 class="l-h4 cp">Latest news &nbsp;<i class="fas fa-chevron-right"></i></h4>
+      <div class="grid-article-2">
+
+        <div class="div-g-1 cp">
+
+          <img src="assets/Images/kreten-1.jpg" alt="" class="slike">
+          <p class="main">Tesla dumped $5B worth of its stock in just 4 days — and traders hate it</p>
+          <p class="main-little">David Canellis4 hours ago</p>
+        </div>
+        <div class="div-g-2 cp">
+          <img src="assets/Images/kreten-2.jpg" alt="" class="slike">
+          <p class="main">Hyrule Warriors: Age of Calamity is the Breath of the Wild prequel we need</p>
+          <p class="main-little">Rachel Kaser4 hours ago</p>
+        </div>
+        <div class="div-g-3 cp">
+          <img src="assets/Images/kreten-3.png" alt="" class="slike">
+          <p class="main">Apple will host the iPhone 12 event on September 15</p>
+          <p class="main-little">Ivan Mehta5 hours ago</p>
+        </div>
+        <div class="div-g-4 cp">
+          <img src="assets/Images/kreten-4.jpg" alt="" class="slike">
+          <p class="main">Warzone is infinitely more fun without vehicles (even though it's just a bug)</p>
+          <p class="main-little">Mix 5 hours ago</p>
+        </div>
+        <div class="div-g-5 cp">
+          <img src="assets/Images/kreten-5.png" alt="" class="slike">
+          <p class="main">Space telescopes are great — so why do we still build ground-based ones?</p>
+          <p class="main-little">The Cosmic Companion 6 hours ago</p>
+        </div>
+        <div class="div-g-6 cp">
+          <img src="assets/Images/kreten-6.png" alt="" class="slike">
+          <p class="main">Turns out Nikola won’t build its Badger hydrogen EV truck after all — GM will</p>
+          <p class="main-little">Matthew Beedhamb6 hours ago</p>
+        </div>
+        <div class="div-g-7 cp">
+          <img src="assets/Images/kreten-7.jpg" alt="" class="slike">
+          <p class="main">If you really want to learn to code, this is a method you should definitely consider</p>
+          <p class="main-little">TNW Deals 6 hours ago</p>
+        </div>
+        <div class="div-g-8 cp">
+          <img src="assets/Images/kreten-8.png" alt="" class="slike">
+          <p class="main">Google blames faulty algorithm for blurring Hong Kong protest graffiti on Street View</p>
+          <p class="main-little">Thomas Macaulay 8 hours ago</p>
+        </div>
+      </div>
+    </article>
+    <!-- LATEST FUNDING PART -->
+    <div class="l1-div">
+      <h4 class="l1-h4">Latest funding rounds on Index.co &nbsp;<i class="fas fa-chevron-right"></i></h4>
+      <p>See more | Get updates by email | Add info</p>
+    </div>
+    <div class="latest-funding">
+      <div class="funding-1">
+        <div class="plus">
+          <img src="assets/Images/kocka.png" alt="" class="jaja">
+
+          <div class="f-1">
+
+            <p>EverCompliant</p>
```

Comment:
- [ ] This needs to be centralized (and occupy more space) on the tablet screen size.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/93134662-702e1780-f6af-11ea-8cc0-aadfd9bdf213.png)|![image](https://user-images.githubusercontent.com/61479648/93134711-820fba80-f6af-11ea-84b6-7e02fd656882.png)  |